app/models/customer.py

```diff
@@ -1,4 +1,31 @@
 from app import db
+from sqlalchemy.sql.functions import func
 
 class Customer(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
\ No newline at end of file
+    customer_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    phone = db.Column(db.String)
+    postal_code = db.Column(db.String)
+    # server_default tells sqlA to pass the default value as part of the CREATE TABLE
+    # func.now() or func.current_timestamp() - they are aliases of each other. This tells DB to calcaate the timestamp itself
+    registered_at = db.Column(db.DateTime(timezone=True), server_default=func.now())
+    #I DON'T KNOW IF THIS IS CORRECT
+    #adding videos attribute to Customer Model
+    videos = db.relationship("Video", secondary="rental", backref="customers")
```

Comment:
The `secondary` attribute can be removed from this relationship. The `secondary` attribute is used with pure join tables (whose sole purpose is just to map out the relationship between two tables). In this case, rental is actually not a pure join table and contains its own columns we want to keep track of ( `due_date` ). 

So in this case, adding the relationship with `backref` to `rental` in both `Video` and `Customer` models would suffice. 

```
rentals = db.relationship("rentals",  backref="customers")
```